packages/slate-react/src/plugin/react-editor.ts

```diff
@@ -978,18 +1022,6 @@ export const ReactEditor: ReactEditorInterface = {
       focusOffset--
     }
 
-    // COMPAT: Triple-clicking a word in chrome will sometimes place the focus
-    // inside a `contenteditable="false"` DOM node following the word, which
-    // will cause `toSlatePoint` to throw an error. (2023/03/07)
-    if (
-      'getAttribute' in focusNode &&
-      (focusNode as HTMLElement).getAttribute('contenteditable') === 'false' &&
-      (focusNode as HTMLElement).getAttribute('data-slate-void') !== 'true'
-    ) {
-      focusNode = anchorNode
-      focusOffset = anchorNode.textContent?.length || 0
-    }
```

Comment:
No longer needed with the changes below and in `ReactEditor.toSlatePoint`.

Reply:
Though I only say this out of intuition; I haven't tested the Chrome triple-clicking situation described in the comment.

Reply:
I should find the PR that added this block and figure out how to replicate the situation to make sure it's still working...

Reply:
Alright, found it was added in #5343, and I found a way to replicate that Chrome triple-click situation. I've confirmed that it still works after this PR's changes. 👍🏼